devtools/conda-envs/test_env.yaml

```diff
@@ -74,3 +77,5 @@ dependencies:
       - sphinx-autobuild
       # no stable release yet with support for serialization
       - MDAnalysis==2.0.0b0
+      # Latest kinoml
+      - https://github.com/openkinome/kinoml/archive/master.tar.gz
```

Comment:
We can't have kinoml installed in this repo's YAML since kinoML will be installed separately again.